src/Service.Tests/Configuration/ComprehensiveHealthReportResponse.json

```diff
@@ -0,0 +1,1357 @@
+{
+  "status": "Healthy",
+  "version": "1.5.0",
+  "app-name": "dab_oss_1.5.0",
+  "configuration": {
+    "rest": true,
+    "graphql": true,
```

Comment:
nit: please update the PR description image of the sample response with the latest fixes for kebab casing